src/features/pastelConf/pastelConf.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs'
+
+import masternodes from './masternodes'
+
+export const updateDefaultPastelConfig = async (
+  pastelConfPath: string,
+): Promise<number> => {
```

Comment:
Instead of a number, can you return something like `Success | Failure | EmptyConfig`

Reply:
I have  updated for this one